hermes/src/api.rs

```diff
@@ -78,7 +83,7 @@ pub async fn run(store: Arc<Store>, mut update_rx: Receiver<()>, rpc_addr: Strin
         .layer(CorsLayer::permissive())
         // non-strict mode permits escaped [] in URL parameters.
         // 5 is the allowed depth (also the default value for this parameter).
-        .layer(Extension(QsQueryConfig::new(false)));
+        .layer(Extension(QsQueryConfig::new(5, false)));
```

Comment:
Might be nice to lift this to a const to keep in sync as I believe you have another 5 from another PR as well. Can place in the root `lib.rs` as a project-wide const.

Reply:
will get this const the next one so I don't have to wait for CI again.